graylog2-server/src/main/java/org/graylog2/alerts/AlertConditionFactory.java

```diff
@@ -34,16 +40,31 @@ public AlertConditionFactory(Map<String, AlertCondition.Factory> alertConditionM
     }
 
     public AlertCondition createAlertCondition(String type,
-                                                Stream stream,
-                                                String id,
-                                                DateTime createdAt,
-                                                String creatorId,
-                                                Map<String, Object> parameters,
-                                                String title) {
+                                               Stream stream,
+                                               String id,
+                                               DateTime createdAt,
+                                               String creatorId,
+                                               Map<String, Object> parameters,
+                                               String title) throws ConfigurationException {
 
         final AlertCondition.Factory factory = this.alertConditionMap.get(type);
         checkArgument(factory != null, "Unknown alert condition type: " + type);
 
+        /*
+         * Ensure the given parameters fulfill the requested configuration preconditions.
+         * Here we strictly use the Configuration object to verify the configuration and don't pass it down to
+         * the factory. The reason for this is that Configuration only support int values, but at least an
+         * alert condition expects a double.
+         */
+        try {
+            final ConfigurationRequest requestedConfiguration = factory.config().getRequestedConfiguration();
+            final Configuration configuration = new Configuration(parameters);
+            requestedConfiguration.check(configuration);
+        } catch (ConfigurationException e) {
+            LOG.error("Could not load alert condition <" + id + ">, invalid configuration detected.");
```

Comment:
The only problem is that it may not be available (titles for alert conditions are only there since 2.1). Anyway, we can add it if available 👍